README.md

```diff
@@ -30,26 +22,67 @@ defp deps do
 end
 ```
 
-### Setup with Plug or Phoenix
+### Setup with Plug and Phoenix
+Capturing errors in Plug applications is done with `Sentry.PlugContext` and `Sentry.PlugCapture`. `Sentry.PlugContext` adds contextual metadata from the current request which is then included in errors that are captured and reported by `Sentry.PlugCapture`.
 
-In your Plug.Router or Phoenix.Router, add the following lines:
+If you are using Phoenix, first add `Sentry.PlugCapture` above the `use Phoenix.Endpoint` line in your endpoint file. `Sentry.PlugContext` should be added below `Plug.Parsers`.
```

Comment:
Nitpick but it may be clearer?

```suggestion
If you are using Phoenix, first add `Sentry.PlugCapture` before the `use Phoenix.Endpoint` line in your endpoint file. `Sentry.PlugContext` should be added below `Plug.Parsers`.
```